src/mito2/src/request.rs

```diff
@@ -563,6 +586,16 @@ pub(crate) struct SenderDdlRequest {
     pub(crate) request: DdlRequest,
 }
 
+#[derive(Debug)]
+pub(crate) enum Command {
```

Comment:
Do we need to wrap it into an enum now? I think it's fine to add this variant to the `WorkerRequest`.